traitsui/testing/tester/_ui_tester_registry/wx/_traitsui/button_editor.py

```diff
@@ -68,6 +68,12 @@ def register(registry):
         handler=lambda wrapper, _: wrapper._target.control.GetLabel()
     )
 
+    registry.register_interaction(
+        target_class=SimpleEditor,
+        interaction_class=IsEnabled,
+        handler=lambda wrapper, _: wrapper._target.control.IsEnabled()
```

Comment:
The fact that you are repeating the same sort of code multiple times here (`lambda wrapper, _: wrapper._target.control.<something>()`) sort of implies that a utility function might be useful here.  Perhaps a something like:
```
def call_control_method(method, *args, **kwargs):
    def handler(wrapper, _):
        bound_method = getattr(wrapper._target.control, method)
        return bound_method(*args, **kwargs)
    return handler
```
Which you can the use as:
```
handler=call_control_method(`IsEnabled`)
```

This design is a lot like the `attrgetter` and `itemgetter` functions from the Python standard library.

Reply:
There are other ways that similar results can be achieved, but they are more magical.

Reply:
Drive-by comment: If you want to go for the above, please choose a name that is more obviously about creating a function that causes side-effect (e.g. `create_handler_for_control_method`), rather than a name that looks like it was going to cause a side-effect.

Reply:
@kitchoi this seems like a useful thing to add to me, I am just not sure the best place for it to sit, as it could be used in other editor implementations for both toolkits.  It would make sense to me to go in a higher level `_interaction_helpers` module but that does not currently exist, and it feels incorrect to put the same function in both toolkit specific `_interaction_helpers` modules.  Any thoughts?  I would be happy to add a module in this PR for it to sit, or we could just open an issue (the scope of this change reaches beyond this PR) and address it in a separate PR.

Reply:
> It would make sense to me to go in a higher level _interaction_helpers module but that does not currently exist, and it feels incorrect to put the same function in both toolkit specific _interaction_helpers modules. Any thoughts? I would be happy to add a module in this PR for it to sit

What you said about a new module makes sense if we are to add this function.
I think the function isn't doing a tons of things and I find the lambda function readable. Considering that it will have to live in a different file, if the new function is readable enough such that others reading the code does not have to find the function to understand what it is doing then it is worth the redirection.  
I will leave this to you whether to do it here.